applications/wg-easy/taskfiles/container.yml

```diff
@@ -0,0 +1,87 @@
+version: "3"
+
+# Development environment tasks
+tasks:
+  build-image:
+    desc: Build development container image
+    vars:
+      DEV_CONTAINER_TAG: '{{.DEV_CONTAINER_TAG | default "latest"}}'
+      CONTAINERFILE: '{{.CONTAINERFILE | default "./container/Containerfile"}}'
+      BUILD_ARGS: '{{.BUILD_ARGS | default ""}}'
+    requires:
+      vars: [DEV_CONTAINER_REPOSITORY, DEV_CONTAINER_IMAGE, CONTAINERFILE]
+
+    cmds:
+      - '{{.CONTAINER_RUNTIME}} build -t {{.DEV_CONTAINER_REPOSITORY}}/{{.DEV_CONTAINER_IMAGE}}:{{.DEV_CONTAINER_TAG}} -f {{.CONTAINERFILE}} .'
+
+  start:
+    desc: Start development container in background
+    silent: true
+    cmds:
+      - task: start-implementation
+
+  # Start development container in background.
+  # It's internal because it's used by shell and start tasks.
+  start-implementation:
```

Comment:
I'm not following why can't this all just be in the "start" task? Task can call and require each other what does having an internal task buy you here if the real start task is just calling the internal one anyway?

Reply:
I've DRY'd this bit some more